tests/smoke/setup_docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+set -x
+sudo sysctl -w vm.max_map_count=262144
+git clone --depth 1 https://github.com/mozilla/addons-server.git
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml pull --quiet
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml up -d --build
```

Comment:
Hmm. [This](https://github.com/mozilla/addons-frontend/blob/e9153c78d8183d61690164692cabb12e88cee313/tests/ui/docker-compose.functional-tests.yml#L8) looks like it will pull the latest `addons-frontend` image from docker hub but we need it to pull [`$CIRCLE_TAG`](https://github.com/mozilla/addons-frontend/blob/e9153c78d8183d61690164692cabb12e88cee313/.circleci/config.yml#L58). Also, this probably needs a wait loop to make sure the image just pushed is available on docker hub before running the tests.

If fixing the tag sounds like a lot of extra work, how about shipping [the old approach](https://github.com/mozilla/addons-frontend/pull/8362/commits/6593c6944848304b9384fd172c85ac9a6fa2f222#diff-1d37e48f9ceff6d8030570cd36286a61L44) just temporarily? The fact that it will block on the dev API isn't such a big deal anymore since broken smoke tests won't cancel the deployment. Just be sure to file an issue to fix it up later.

Reply:
It won't as the ```tests/ui/docker-compose.functional-tests.yml``` is instructed to use an image that has to be built.

Reply:
I guess if you go back to building and testing the local image then you wouldn't need a dockerhub wait loop.

Reply:
I don't think that wait is needed anymore. I think the image was a bit slow to start when I implemented it, but its better now

Reply:
Ah wait, I misunderstood. I am using ```addons-frontend``` instead of ```mozilla/addons-frontend``` because the docker build step of the config builds an image named ```addons-frontend``` and if that doesn't exist it will build it.